docs/src/contributing-to-popochiu/basics.md

```diff
@@ -0,0 +1,105 @@
+
+Attempting to accommodate every possible use case is neither practical nor desirable. A framework that tries to do everything, in every possible way, becomes overly complex to use, difficult to learn, and costly to maintain.
+
+If you wish to contribute a structural change, an alternative to an existing feature, or support for workflows that deviate from those already provided, the best approach is to first open an issue with a clear proposal (or check if it’s already in the backlog). This allows for a productive discussion with maintainers and the community before submitting a pull request with significant work already done.
+
+By discussing your idea, we can provide a broader perspective and suggest a course of action that aligns with future developments. More often than not, we’ll simply validate your proposal and eagerly await your code with open arms.
+
+### Always assume competence, with a critical stance
+
+Ecco una versione sintetizzata e leggermente riformulata per maggiore chiarezza e impatto:
+
+---
+
+As Joel Spolsky aptly noted in his classic post [Things You Should Never Do, Part 1](https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/), **_“It’s harder to read code than to write it.”_** Understanding *what* code does is challenging enough, but grasping *why* it was written that way can be even harder.
+
+When exploring Popochiu’s code for the first time, you might find it difficult to navigate and feel that many things were done “wrong.” Rest assured, however, that the architecture reflects careful reasoning, analysis, and - of course - a bit of personal taste. This applies to core as well as contributed code.
+
+Is it perfect? Certainly not. Popochiu began as a solo project on Godot 3.5, and over time, some original decisions have shown their limits. Godot itself evolves, offering new possibilities that outshine older approaches. While some choices hold up well, others are due for improvement.
+
+How can you distinguish between deliberate decisions and past missteps? Communication is key. Reach out to the maintainers on Discord or open an issue to discuss your ideas with the community. What seems like a poor choice might make sense in a broader context. Or it might confirm your instincts, giving you the opportunity to improve things.
+
+By collaborating and discussing upfront, you can ensure your contribution is impactful and warmly welcomed!
+
+### Leave the place better than you found it
+
+We strive to build upon design patterns and best practices, keeping the code clean and readable at all times.
+
+Adventure games are so simple that they used to captivate us on a 286: running one on modern hardware - even a portable console or a phone - is a breeze. This allows us to focus on the performance that really matters: **how efficiently people can read, write, and maintain Popochiu's code**. By prioritizing clarity and maintainability, we make the codebase more stable and predictable for everyone.
```

Comment:
Just out of curiosity. What does the 286 means? Maybe I'm slow today, but I didn't get it 🤓 .

Reply:
Referring to this: https://en.wikipedia.org/wiki/Intel_80286

But maybe this works better if I name a Commodore 64 or an Amiga 500?